public/app/plugins/panel/canvas/Connections.tsx

```diff
@@ -192,7 +192,9 @@ export class Connections {
   render() {
     return (
       <>
-        <ConnectionAnchors setRef={this.setConnectionAnchorRef} handleMouseLeave={this.handleMouseLeave} />
+        {this.scene.shouldEnableConnections && (
```

Comment:
Any reason we need more control than this:
```suggestion
        {this.scene.isEditingEnabled && (
```

Reply:
Currently they are enabled only while editing. Maybe we should've talked more about this, but I think this is a cleaner ui if you don't want to deal with connections. I believe draw.io has a checkbox to enable/disable them.

Reply:
oh I see, they do.  I am reluctant to add too many options while we still figure out what is what.

Reply:
Fair point. We can put this on hold until we figure out if we need it or not.